Source/utils/stubs.h

```diff
@@ -1,23 +1,21 @@
 #pragma once
 
-#include <cassert>
-#include <cstdio>
-#include <cstdlib>
+#include "utils/log.hpp"
 
-#define UNIMPLEMENTED()                                                         \
-	do {                                                                        \
-		SDL_Log("UNIMPLEMENTED: %s @ %s:%i", __FUNCTION__, __FILE__, __LINE__); \
-		abort();                                                                \
+#define UNIMPLEMENTED()                                                                        \
+	do {                                                                                       \
+		::devilution::LogDebug("UNIMPLEMENTED: {} @ {}:{}", __FUNCTION__, __FILE__, __LINE__); \

```

Comment:
Again, this seems completely unrelated to Byte -> uint8 conversion cleanup to me.

Reply:
See commit message - cyclical include chain without this change (and we want each commit to be buildable for easier bisects)

Reply:
Wouldn't it be possible to make these changes before the byte change, then? The fact they are "needed" doesn't preclude them being done _prior_ as their own commit IMHO.

Reply:
Yeah it'd certainly be cleaner in its own commit but extra work for little gain -- it's not difficult to review this PR.